docs/user/dashboard/tsvb.asciidoc

```diff
@@ -17,15 +18,19 @@ Open *TSVB*, then make sure the required settings are configured.
 
 . On the *New visualization* window, click *TSVB*.
 
-. In *TSVB*, click *Panel options*, then make sure the following settings are configured:
+. In *TSVB*, click *Panel options*, then specify the required *Data* settings.
 
-* *Index pattern*
-* *Time field*
-* *Interval*
+.. From the *Index pattern* dropdown, select the index pattern you want to visualize. 
++
+To visualize an {es} index, open the *Index pattern select mode* menu, deselect *Use only {kib} index patterns*, then select the {es} index from the dropdown.
+
+.. From the *Time field* dropdown, select the field you want to visualize, then enter the field *Interval*.
 
-. Select a *Drop last bucket* option. It is dropped by default because the time filter intersects the time range of the last bucket, but can be enabled to see the partial data.
+.. Select a *Drop last bucket* option. 
++
+By default, *TSVB* drops the last bucket because the time filter intersects the time range of the last bucket. To view the partial data, select *No*.
```

Comment:
this paragraph is invalid for 7.14 because of #97257

Reply:
Thanks for the feedback! I'll open a separate PR to make this change for 7.14. I added the 7.14 label to this PR to indicate I'm backporting to 7.x.